gorush/notification_apns.go

```diff
@@ -2,17 +2,33 @@ package gorush
 
 import (
 	"errors"
+	"fmt"
 	"path/filepath"
 	"time"
 
 	apns "github.com/sideshow/apns2"
 	"github.com/sideshow/apns2/certificate"
 	"github.com/sideshow/apns2/payload"
+	"github.com/sideshow/apns2/token"
 )
 
 // InitAPNSClient use for initialize APNs Client.
 func InitAPNSClient() error {
 	if PushConf.Ios.Enabled {
+
+		if PushConf.Ios.P8Key != "" {
```

Comment:
Adding PushConf.Ios.KeyType could make it easier to switch w/o deleting P8K config

Reply:
the main problem with that approach in my mind is it takes away the flexibility of providing a file and just using the extension to determine key type. Because technically, if we provide a .p8 file it could still work (I haven't included it in this PR, maybe I should)